lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -21,6 +21,9 @@ module Style
       #
       # NOTE: Regexp and Range literals are frozen objects since Ruby 3.0.
```

Comment:
It seems we are using different style to highlight notes. I went with `NOTE: ` but happy to use "From Ruby 3.0, ..." pattern as seen above.

Reply:
`NOTE:` is better.